dotnet/src/Planners/Planners.Handlebars/Handlebars/Exceptions/HandlebarsPlanCreationException.cs

```diff
@@ -0,0 +1,37 @@
+﻿// Copyright (c) Microsoft. All rights reserved.
+
+namespace Microsoft.SemanticKernel.Planning.Handlebars;
+
+/// <summary>
+/// Provides extension methods for rendering Handlebars templates in the context of a Semantic Kernel.
+/// </summary>
+internal sealed class HandlebarsPlanCreationException : KernelException
```

Comment:
Should this be internal? Are we not expecting clients to catch this?

Reply:
Good catch - made the class public and added XML comments